app/models/checkout.rb

```diff
@@ -10,4 +11,11 @@ class Checkout < ApplicationRecord
   def due_date
     self.created_at + CHECKOUT_PERIOD_IN_DAYS.days
   end
+
+  def fine
+    days_overdue = (Time.zone.now - self.due_date).to_i/1.day
```

Comment:
Also, we generally avoid the explicit use of `self` unless it is required. For instance, if you did `due_date = Date.Today`, you would need `self` to disambiguate between creating and assigning a local variable named `due_date` and calling the instance method `due_date=`. But here, you do not need `self`.